iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalCatalog.java

```diff
@@ -57,6 +63,18 @@ protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
         new MetricsReporter(this.meterRegistry, METRICS_PREFIX, Lists.newArrayList()));
   }
 
+  /** Overwritten for annotation purpose. */
+  @Override
```

Comment:
we override the same method on li-openhouse side, do you anticipate the code working ok there?

Reply:
hmm thanks for catching that, I didn't realize this is being overwritten. I will need to test with that then.

Reply:
```
    public Table createTable(TableIdentifier identifier, Schema schema, PartitionSpec spec, String location, Map<String, String> properties) {
        log.info("Creating table {} at {}", identifier.toString(), location);
        properties.put(TableProperties.WRITE_METADATA_LOCATION, String.format("%s/%s", location, METADATA_SUB_FOLDER));
        properties.put(TableProperties.WRITE_DATA_LOCATION, String.format("%s/%s", location, DATA_SUB_FOLDER));
        // TODO: Safe to remove TableProperties.WRITE_FOLDER_STORAGE_LOCATION from properties when clients
        //  have upgraded.
        properties.put(TableProperties.WRITE_FOLDER_STORAGE_LOCATION,
            String.format("%s/%s", location, DATA_SUB_FOLDER));

        return super.createTable(identifier, schema, spec, location, properties);
    }
```

What I anticipated is the `super.createTable` call will trigger this annotation aspect.

Reply:
Ack! let's ensure we test this out particularly.